underpants.go

```diff
@@ -247,6 +267,22 @@ func urlFor(scheme, host string, r *http.Request) *url.URL {
   return &u
 }
 
+func userMemberOf(c *conf, u *user, groups []string) bool {
+  for _, group := range groups {
+    if group == "*" {
+      return true
+    }
+
+    for _, allowedUser := range c.Groups[group] {
```

Comment:
Yeah, we've only a half dozen groups or so at the moment and <100 users total.  Totally cool if you want to make it constant time.